chef_master/source/chef_client_overview.rst

```diff
@@ -15,7 +15,8 @@ Chef Infra Client Overview
 
      - .. tag chef_client_summary
 
-       Chef Infra Client is an agent that runs locally on every node that is under management by Chef Infra Server. When a Chef Infra Client is run, it will perform all of the steps that are required to bring the node into the expected state, including:
+       Chef Infra Client is an agent that runs locally on every node that is under management by Chef Infra Server. The chef-client install package includes a self-contained ruby install
```

Comment:
I'm not sure about leading with this. Our first introduction to what the client is shouldn't be that it ships with Ruby and everything else. That's getting pretty deep into the weeds right off the bat.

Reply:
It is in the weeds and we need a specific place for "Chef Infra Client contains the following: X, Y, Z, ruby install". Who else thinks the content of this "overview" page is all over the place?

Reply:
@tas50 @sean-horn Can you give me a list of the important things included in the chef-client? For example, the self-contained ruby install. The versions of said things might also be important. Thanks!

Reply:
@kagarmoe Feel free to place it wherever makes sense. This was a sales/field request. I don't think we should go into versions(they change) nor any deeper than what I've written there. Folks don't necessarily need/want to know. Those that do need to know will get there fairly easily.

However, it could look something like 
* A chef-client OS package produced by Omnibus https://github.com/chef/omnibus encapsulates the following and more
  * A Ruby interpreter built from source (not in windows)
  * Various other software dependencies from https://github.com/chef/omnibus-software included in the build

Reply:
I'm putting this onto the "Install ChefDK" page.